features/step_definitions/electionSteps.js

```diff
@@ -18,6 +18,22 @@ module.exports = function() {
         this.election.save(callback);
     });
 
+    this.Given(/^I am on the retrieve vote page$/, function(callback) {
+        browser.get('/retrieve');
+        callback();
+    });
+
+    this.Given(/^I enter a random string as vote hash$/, function(callback) {
```

Comment:
If #103 gets merged first you can use https://github.com/webkom/vote/pull/103/files#diff-f42390971b8bdd5b2f707a60adbf8b6fR89 - `I fill in model "voteHash" with "notahash"`